contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeOptions.vue

```diff
@@ -271,75 +309,16 @@
         // Otherwise, don't do anything
         return () => {};
       },
-      editTitleDescription() {
-        this.trackAction('Edit title and description');
-        this.openQuickEditModal({
-          modal: QuickEditModals.TITLE_DESCRIPTION,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editTags() {
-        this.trackAction('Edit tags');
-        this.openQuickEditModal({
-          modal: QuickEditModals.TAGS,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editLanguage() {
-        this.trackAction('Edit language');
-        this.openQuickEditModal({
-          modal: QuickEditModals.LANGUAGE,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editCategories() {
-        this.trackAction('Edit categories');
-        this.openQuickEditModal({
-          modal: QuickEditModals.CATEGORIES,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editLevels() {
-        this.trackAction('Edit levels');
-        this.openQuickEditModal({
-          modal: QuickEditModals.LEVELS,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editLearningActivities() {
-        this.trackAction('Edit learning activities');
-        this.openQuickEditModal({
-          modal: QuickEditModals.LEARNING_ACTIVITIES,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editSource() {
-        this.trackAction('Edit source');
-        this.openQuickEditModal({
-          modal: QuickEditModals.SOURCE,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editAudience() {
-        this.trackAction('Edit audience');
-        this.openQuickEditModal({
-          modal: QuickEditModals.AUDIENCE,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editCompletion() {
-        this.trackAction('Edit completion');
-        this.openQuickEditModal({
-          modal: QuickEditModals.COMPLETION,
-          nodeIds: [this.nodeId],
-        });
-      },
-      editWhatIsNeeded() {
-        this.trackAction('Edit what is needed');
-        this.openQuickEditModal({
-          modal: QuickEditModals.WHAT_IS_NEEDED,
-          nodeIds: [this.nodeId],
-        });
+      quickEditModalFactory(modal) {
+        return $event => {
+          this.openQuickEditModal({
+            modal,
+            nodeIds: [this.nodeId],
+          });
+          const trackActionLabel = modal.replace(/_/g, ' ').toLowerCase();
+          this.trackAction(`Edit ${trackActionLabel}`);
```

Comment:
Just checking this is not string that requires translation?

Reply:
Oh no, these are strings just for analytics purposes.